hawkular-inventory-paths/pom.xml

```diff
@@ -28,6 +28,13 @@
 
   <dependencies>
     <dependency>
+      <groupId>io.swagger</groupId>
+      <artifactId>swagger-annotations</artifactId>
+      <version>${version.io.swagger}</version>
```

Comment:
@pavolloffay could you please answer that question? I just used the version of `io.swagger` that you introduced in commit https://github.com/hawkular/hawkular-inventory/commit/5667f30a35fcd5fb4b1d6d43bb00cdc91e64a1ee..